tests/test_trusted_metadata_set.py

```diff
@@ -240,12 +240,14 @@ def root_new_version_modifier(root: Root) -> None:
         self.trusted_set.update_root(root)
 
     def test_update_root_new_root_fail_threshold_verification(self) -> None:
-        # new_root data with threshold which cannot be verified.
-        root = Metadata.from_bytes(self.metadata[Root.type])
-        # remove root role keyids representing root signatures
-        root.signed.roles[Root.type].keyids.clear()
```

Comment:
Not really relevant anymore, but I don't understand why the old test didn't do `root.signatures.clear()` here.